src/gigs-board/components/posts/Post.jsx

```diff
@@ -86,7 +87,7 @@ const childPostIdsUnordered =
 
 const childPostIds = props.isPreview ? [] : childPostIdsUnordered.reverse();
 const expandable = props.isPreview ? false : props.expandable ?? false;
-const defaultExpanded = expandable ? props.defaultExpanded : true;
+const defaultExpanded = expandable ? props.defaultExpanded : false;
```

Comment:
Is this intentional?

Reply:
yes. was intended to keep the reply posts cleaner.